lib/newspaper_works/ingest/chronam_publication_info.rb

```diff
@@ -0,0 +1,133 @@
+require 'faraday'
+require 'nokogiri'
+
+module NewspaperWorks
+  module Ingest
+    # Publication info from ChronAm as remote authority for metadata
+    class ChronAmPublicationInfo < BasePublicationInfo
+      attr_accessor :issn, :title, :place_name, :place_of_publication, :language
+
+      XML_NS = {
+        dcterms: 'http://purl.org/dc/terms/',
+        frbr: 'http://purl.org/vocab/frbr/core#',
+        owl: 'http://www.w3.org/2002/07/owl#',
+        rda: 'http://rdvocab.info/elements/',
+        rdf: 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+        rdfs: 'http://www.w3.org/2000/01/rdf-schema#'
+      }.freeze
+
+      BASE_URL = 'https://chroniclingamerica.loc.gov/lccn'.freeze
+
+      def initialize(lccn)
+        # true until loaded
+        @empty = true
+        super(lccn)
+        @issn = nil # chronam doesn't have this
+      end
+
+      def empty?
+        @empty
+      end
+
+      def inspect
+        format(
+          "<#{self.class}:0x000000000%<oid>x " \
+            "\tlccn: '#{@lccn}'>",
+          oid: object_id << 1
+        )
+      end
+
+      def load_place
+        place_match = find('//rda:placeOfPublication')
+        return if place_match.nil?
+        @place_name = place_match.first.text
+        @place_of_publication = NewspaperWorks::Ingest.geonames_place_uri(
```

Comment:
For indexing to work as expected, the `place_of_publication` value should be set as a `Hyrax::ControlledVocabularies::Location` object, not `String.` Something like this should work:
```
def load_place
  place_match = find('//rda:placeOfPublication')
  return if place_match.nil?
  @place_name = place_match.first.text
  uri = NewspaperWorks::Ingest.geonames_place_uri(@place_name)
  @place_of_publication = Hyrax::ControlledVocabularies::Location.new(uri)
end
```
Specs will also need to be adjusted accordingly.